### PR TITLE
✨ Pull Request

### PR DESCRIPTION
**Pull request by aniknia**
📓 Referenced Issue

Partly addresses issue https://github.com/responsively-org/responsively-app/issues/667 and adresses the issue with https://github.com/responsively-org/responsively-app/pull/750
ℹ️ About the PR

Pulled information for IPhone 14, 14 Pro, 14 Plus, and 14 Pro Max from [apple](https://www.apple.com/iphone-14/specs/) and [blisk.io](https://blisk.io/devices)

### CODE REVIEW

**Comment by aniknia** [comment]
@manojVivek does this correct what you mentioned [here](https://github.com/responsively-org/responsively-app/pull/750)

File: desktop-app/src/common/deviceList.ts [hunk 1]
```diff
@@ -156,7 +192,7 @@ export const defaultDevices: Device[] = [
     isMobileCapable: true,
   },
   {
-    id: '10011',
+    id: '10015',
```
Comment (manojVivek) [inline desktop-app/src/common/deviceList.ts:195]: Please don't modify the ids of existing devices, append the new devices to the end of the apple products in this list.

File: desktop-app-legacy/app/constants/chromeEmulatedDevices.js
Comment (manojVivek) [inline desktop-app-legacy/app/constants/chromeEmulatedDevices.js:1]: Also, remove the changes to this file since it's no longer needed.

**Review by manojVivek — CHANGES_REQUESTED** [review]
Thanks for the PR. Requested a couple of changes and then it's good for merge.

File: desktop-app/src/common/deviceList.ts (hunk 1 above)
Comment (aniknia) [reply desktop-app/src/common/deviceList.ts]: I made the changes

File: desktop-app-legacy/app/constants/chromeEmulatedDevices.js
Comment (aniknia) [reply desktop-app-legacy/app/constants/chromeEmulatedDevices.js:1]: I made the changes

**Review by manojVivek — APPROVED** [review]
Thanks so much for adding these! Looking forward to more such contributions.

**Comment by manojVivek** [comment]
@all-contributors Please add @aniknia for code.

**Pull request merged**